### DISCUSSION TITLE
USSD icon

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#2548] [Request: Icon] [answered]
1. Link of icon in Streamline: https://www.streamlinehq.com/icons/streamline-light/phones-mobile-devices/content-actions/phone-action-text-1

2. Twilio.org is creating a new "product" called USSD (it's like SMS but it's popular in lesser developed countries; unlike SMS, the recipient will text, for instance, # 1234*345 #, they will see a menu with selections to choose from - so no free form worded responses)

We'll need an icon for the product to show in the product explore and sidenav:

![Screen Shot 2022-07-21 at 3 10 24 PM](https://user-images.githubusercontent.com/18605252/180296791-c8743412-f44c-4578-af62-2044c02b15dd.png)
![Screen Shot 2022-07-21 at 3 12 44 PM](https://user-images.githubusercontent.com/18605252/180297143-748e4482-04c4-4a3b-908e-9b786ca1d6ee.png)

3. This is a UI icon

Not sure how to get the SVG since it's a pro feature?

**Comment by shleewhite** [comment]
Hi @pamelahu!

Since this is a new product icon, it needs to go through Brand first. You can reach out to them in the #mkg-brand-corecreative slack channel and follow [their process](https://www.twilio.com/brand/documentation/creative-project-request#How_to_submit_a_new_request).

**Reply by kate-porter** [reply]
Hey @pamelahu, just wanted to check in here to see if you were able to connect with Brand about creating this icon.

**Comment by loreina** [comment]
@pamelahu Just following up from Slack. I'm going to remove the "Active Discussion" label for now. Feel free to reply here when Brand has the icon ready and we'll make a ticket in our backlog to add the icon to Paste.

**Reply by pamelahu** [reply]
@loreina Hey, the Brand team has finished the icon. What steps should I take to add the icon to Paste?

**Reply by loreina** [reply]
@pamelahu Hey, I just created a branch in Figma and added you as a reviewer. Once that's published in Figma, the team will fast-follow with PR to code and send an update here.

**Comment by gloriliale** [comment, marked as answer]
Hi @pamelahu!

The USSD icon is available in the latest version of Paste!
